site/components/header/small-screen-nav.js

```diff
@@ -29,6 +29,9 @@ export default class SmallScreenNav extends React.Component {
   render() {
     const { navOpen } = this.state;
     const navTabIndex = navOpen ? 0 : -1;
+    const { stateNavigator } = this.context;
+    const contactUsLink = `${stateNavigator.historyManager.getHref(stateNavigator.getNavigationLink('homePage'))}#ContactUs`;
```

Comment:
@JoeStanton We went for a solution that worked without JavaScript. We did investigate putting something in if JavaScript was on, to avoid the full page reload, but it was getting a bit messy. I'm happy to take another look